packages/context/test/unit/context.ts

```diff
@@ -4,7 +4,8 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {expect} from '@loopback/testlab';
-import {Context, Binding} from '../../src';
+import {Context, Binding, inject, ValueProvider} from '../../src';
+import {MockValueProvider, MockFunctionProvider, MockConstructorProvider, MockLoopbackController} from './fixtures/mock-binding-providers';
```

Comment:
-1 for mocks defined outside of the test file that uses them. It makes it more difficult to understand what a test does, because one has to read multiple files. Also a mock should be narrowly focused to provide just enough functionality need by the test that uses it and thus shouldn't be shared in most cases.